noir_stdlib/src/meta/op.nr

```diff
@@ -19,3 +19,74 @@ impl UnaryOp {
         self.op == 3
     }
 }
+
+struct BinaryOp {
+    op: Field
+}
+
+impl BinaryOp {
+    fn is_add(self) -> bool {
+        self.op == 0
```

Comment:
Worth creating some globals for this for readability?

Reply:
Hey Tom, welcome back! 👋 

I'm not sure. I think it's clear what the number means based on the enclosing method name. But if you think it's worth it, let me know and I'll do it.

Reply:
I don't have overly strong opinions on this. Based on the number of these that we have, I would add these as a quasi-enum but it's 2-1 on this point.